### PR TITLE
feat: adding navbar

### LINKED ISSUE #1: Create Navbar
Implement the navigation bar for the landing page. It should be 
- responsive 
- links to different sections.
- show MLH banner
- blur and stick to top as you scroll down

### PR DESCRIPTION
**Pull request by demanr**
PR for navbar. All socials links work, and clicked the DeltaHacks logo should lead to home. Only navigation to FAQ works as other areas are not implemented yet, but it is a simple change. Upon md or smaller screen size hamburger menu alternative appears, which upon opening will stay open unless it is clicked out of or a position is chosen (ie if the screen size is increased greater than md, the window will remain open. this can be changed)

### CODE REVIEW

**Review by Krish120003 — DISMISSED** [review]

**Review by Krish120003 — DISMISSED** [review]

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+          </Drawer.Portal>
+        </Drawer.Root>
+      </div>
+    </div>
+  );
+};
+
+export default Navbar;
+
+//  <div className="drawer drawer-end">
```
Comment (kabirsg) [inline src/components/Navbar.tsx]: @demanr Did you want to keep these comments in the push?

**Review by kabirsg — COMMENTED** [review]
Overall, looks good other than the extra comments

File: src/components/Navbar.tsx (hunk 1 above)
Comment (demanr) [reply src/components/Navbar.tsx]: I'll remove them then

Comment (kabirsg) [reply src/components/Navbar.tsx]: Ok that works

**Review by kabirsg — APPROVED** [review]
LGTM

**Pull request merged**